### PR TITLE
simplify report closure

### PR DESCRIPTION
**Pull request by vishalpandeyvip**

### CODE REVIEW

**Review by labhvam5 — APPROVED** [review]

File: src/app/core/models/db/workspace.model.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export type Workspace = {
   fyle_org_id: string;
   xero_short_code: string;
   last_synced_at?: Date;
+  ccc_last_synced_at?: Date;
```
Comment (ashwin1111) [inline src/app/core/models/db/workspace.model.ts:13]: ? is not required

File: src/app/shared/components/configuration/export-settings/export-settings.component.ts [hunk 2]
```diff
@@ -269,20 +268,49 @@ export class ExportSettingsComponent implements OnInit, OnDestroy {
...
+
+    const getSettingsData: (Observable<ExportSettingGet> | Observable<GroupedDestinationAttribute> | Observable<WorkspaceGeneralSetting>)[]= [
       this.exportSettingService.getExportSettings(),
       this.mappingService.getGroupedXeroDestinationAttributes(destinationAttributes)
-    ]).subscribe(response => {
-      this.exportSettings = response[0];
-      this.bankAccounts = response[1].BANK_ACCOUNT;
+    ];
+
+    if (!this.isOnboarding){
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/export-settings/export-settings.component.ts:277]: one space after )

File: src/app/shared/components/configuration/export-settings/export-settings.component.ts [hunk 3]
```diff
@@ -269,20 +268,49 @@ export class ExportSettingsComponent implements OnInit, OnDestroy {
...
+      getSettingsData.push(this.workspaceService.getWorkspaceGeneralSettings());
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/export-settings/export-settings.component.ts:278]: why is this API call required when we can get `is_simplify_report_closure_enabled` from the export_settings call?

File: src/app/shared/components/configuration/export-settings/export-settings.component.ts [hunk 4]
```diff
@@ -269,20 +268,45 @@ export class ExportSettingsComponent implements OnInit, OnDestroy {
...
     const destinationAttributes = ['BANK_ACCOUNT', 'TAX_CODE'];
+
     forkJoin([
       this.exportSettingService.getExportSettings(),
       this.mappingService.getGroupedXeroDestinationAttributes(destinationAttributes)
     ]).subscribe(response => {
       this.exportSettings = response[0];
       this.bankAccounts = response[1].BANK_ACCOUNT;
+      
+      if (!this.isOnboarding) {
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/export-settings/export-settings.component.ts:279]: remove this, this is not required

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**